docker/Dockerfile-gbpserver

```diff
@@ -0,0 +1,14 @@
+FROM alpine:3.7
```

Comment:
At some point we should consider probably move this to 3.9 (our other base images are all 3.9).